setup.cfg

```diff
@@ -1,19 +1,32 @@
 [flake8]
 ignore =
     # Conflicts with black
-    E203
+    E203 
     # Line break occurred before a binary operator. Update by W504 Line
     W503
     # Allow "import torch.nn.functional as F"
     N812
+    # Line length, handled separately by max-line-length
+    E501
+    # too many leading '#' for block comments
+    E266
+    
 
 per-file-ignores =
     # Unused imports in __init__.py are OK
     **/__init__.py:F401
     
 exclude = 
     *_pb2*,
+    .git,
+    __pycache__,
+    build,
+    dist,
+    .venv
 
-max-line-length = 100
+max-line-length = 120
```

Comment:
Any specific reason to increase?

Reply:
standard max-line-length used from 79 or 120. so increased to 120. if not needed we can revert it

Reply:
If 100 returns lint errors, it is better to trim those lines to be under 100 chars instead.

Reply:
reverted to 100